src/Parser/TemplateParser.cs

```diff
@@ -94,7 +95,7 @@ private void ParseCloseBracketPart()
         private void ParseHole(HoleType type)
         {
             string name = ReadUntil(HoleDelimiters);
-            string aligment = Peek() == ',' ? ParseAlignment() : null;
+            int? aligment = Peek() == ',' ? ParseAlignment() : (int?)null;
```

Comment:
if this is an `if else`, you don't need the cast. But it this also more performant? (as casts are "slow")